website/content/en/highlights/2022-03-22-0-21-0-upgrade-guide.md

```diff
@@ -222,6 +223,19 @@ If `.value` was zero, Vector would panic. This can be fixed by handling the erro
 .remainder = 50 % .value ?? 0
 ```
 
+#### AWS SDK Migration {#aws-sdk-migration}
+
+We are in the process of migrating sources and sinks that utilize AWS to the official AWS SDK (from Rusoto).
+This comes with some benefits such as support for IMDSv2. This requires us to deprecate some config options.
+
+If you were providing a custom path to an AWS credentials file, this is no longer supported for components that have
+been migrated. It is still possible to use a credentials file, but it must be stored in the default location (`~/.aws/credentials`)
+
+This affects the following components:
```

Comment:
Would we want to mention the environment variables that can be used to control where it looks for the static config and credentials files? `AWS_SHARED_CREDENTIALS_FILE` and `AWS_CONFIG_FILE`?

Reply:
Or maybe this, which is more specific to the SDK we are using: https://docs.aws.amazon.com/sdk-for-rust/latest/dg/credentials.html